plutus-use-cases/src/Plutus/Contracts/Uniswap/OffChain.hs

```diff
@@ -489,7 +489,7 @@ ownerEndpoint :: Contract (Last (Either Text Uniswap)) EmptySchema ContractError
 ownerEndpoint = do
     e <- mapError absurd $ runError start
     tell $ Last $ Just e
-    void $ waitNSlots 10
+    void $ waitNSeconds 10
```

Comment:
honestly, these calls seem kind of suspicious. Probably they should be waiting for something!